src/components/ContactList/NoContact.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import Header from '../Header/Header'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components'
+
+export default function NoContact() {
```

Comment:
The name "NoContact" is a bit missleading in my opinion. I would suggest something like "ZeroContacts" or "NoContactsYet".